classes/form/CustomerAddressForm.php

```diff
@@ -86,6 +86,10 @@ public function validate()
             }
         }
 
+        if ($hookReturn = Hook::exec('actionValidateCustomerAddressForm', array('form' => $this)) != '') {
+            $is_valid &= $hookReturn;
```

Comment:
Maybe you should handle unexpected type of value by adding a (bool) cast before `$hookReturn` ?